README.md

```diff
@@ -21,16 +44,23 @@ path = /{path-to-spotify-heroic-datasource}
 If using Docker, you can mount this repo into the Grafana plugin directory:
 
 ```
-docker run -it -p 3000:3000 --name=grafana -v `pwd`:/var/lib/grafana/plugins/spotify-heroic-datasource grafana/grafana
+docker run -it -p 3000:3000 --name=grafana -v `pwd`/dist:/var/lib/grafana/plugins/spotify-heroic-datasource grafana/grafana
 ```
 
-Can also be installed with grafana-cli
 
-`grafana-cli plugins install spotify-heroic-datasource`
+# Releasing
+
+
+1. In a terminal, navigate to the repo, ensure you are on the master branch and then run...
+1. git pull --rebase origin master
+1. git log (ensure you are on the latest version)
+1. git fetch --tags
+1. git tag -l (this lists the previous release tags. increment the tag for your release and use semantic versioning)
+1. git tag ${tag}
+1. git push origin ${tag}
+1. Take the commit hash made for the tag released and append it to [repo.json](https://github.com/grafana/grafana-plugin-repository/blob/master/repo.json).
```

Comment:
This is already being built by travis right? I think @hexedpackets set that up. If it isn't already, is there any way to just get Travis to handle the git tags so that all you need to do to update the upstream repo is grab the commit hash of the latest Travis build?

Reply:
that's a better idea - i need to check how we can get travis to bump the version 👍 

and also patch the version in https://github.com/spotify/spotify-heroic-datasource/blob/master/package.json#L3